packages/react-scripts/scripts/eject.js

```diff
@@ -129,7 +129,7 @@ prompt(
   // Add Jest config
   console.log('  Adding ' + cyan('Jest') + ' configuration');
   appPackage.jest = createJestConfig(
-    filePath => path.join('<rootDir>', filePath),
+    filePath => path.posix.join('<rootDir>', filePath),
```

Comment:
No, it doesn't, but we always pass it `/`, and path.win32 switches those to `\`.
See https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/utils/createJestConfig.js#L24.